src/main/java/uk/gov/companieshouse/web/emergencyauthcodeweb/controller/eac/ListOfOfficersController.java

```diff
@@ -89,7 +91,12 @@ public String postListOfOfficers(@PathVariable String requestId,
         // Retrieve company number from API for later request
         String companyNumber;
         try {
-            companyNumber = emergencyAuthCodeService.getEACRequest(requestId).getCompanyNumber();
+            EACRequest eacRequest = emergencyAuthCodeService.getEACRequest(requestId);
+            companyNumber = eacRequest.getCompanyNumber();
+            if (SUBMITTED_STATUS.equals(eacRequest.getStatus())) {
```

Comment:
I think it would be more appropriate if we did this below because we're checking if the status of the actual eacRequest is submitted

```
if(eacRequest.getStatus().equals(SUBMITTED_STATUS))
```

Reply:
> I think it would be more appropriate if we did this below because we're checking if the status of the actual eacRequest is submitted
> 
> ```
> if(eacRequest.getStatus().equals(SUBMITTED_STATUS))
> ```

Done